handlers/oam_handler.go

```diff
@@ -95,7 +95,8 @@ func (h *Handler) PatternFileHandler(
 	if h.kubeclient.DynamicKubeClient == nil {
 		kc, err := meshkube.New(prefObj.K8SConfig.Config)
 		if err != nil {
-			logrus.Error("failed to create kube client: ", err)
+			h.log.Error(ErrKubeClient(err))
+			http.Error(w, ErrKubeClient(err).Error(), http.StatusInternalServerError)
 			rw.WriteHeader(http.StatusInternalServerError)
 			fmt.Fprintf(rw, "Error:%s", err)
```

Comment:
The line number 99 is equivalent to line 100 and 101 so I think we can comment out the line 100 and 101